src/udl/skel/PHP/pylib/lang_php.py

```diff
@@ -2567,6 +2569,39 @@ def _getArgumentsFromPos(self, styles, text, pos):
                     phpArgs.append(phpArg)
         return phpArgs, p
 
+    def _getFuncTypeHintFromPos(self, styles, text, pos):
```

Comment:
This comment isn't for you:
It's confusing that `text` here means a line of code and later it is an array of words.  Best to do as you have done though and keep with the convention of the existing code.